recipes/qhull/all/patches/fix-cmake-7.3.2.patch

```diff
@@ -1,38 +1,120 @@
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -333,7 +333,7 @@ set(
+@@ -333,30 +333,29 @@ set(
          index.htm
  )
  
 -include_directories(${CMAKE_SOURCE_DIR}/src)
+-
+-if(CMAKE_BUILD_TYPE MATCHES "[dD]ebug")
```

Comment:
why not follow upstream naming scheme?

Reply:
Because it breaks for multi-configuration generators: https://github.com/qhull/qhull/pull/76